Resources/Prototypes/Recipes/Crafting/Graphs/storage/lockergeneric.yml

```diff
@@ -0,0 +1,36 @@
+- type: constructionGraph
+  id: LockerGeneric
+  start: start
+  graph:
+    - node: start
```

Comment:
style is to not indent - an extra level
```
graph:
- node: start
  edges:
  - to: done
    ...
```
etc

Reply:
I literally copied this code!

Reply:
yes and you have to update legacy code if you copy it

Reply:
pro resolve